exe/node/Main.hs

```diff
@@ -131,7 +131,8 @@ main = do
     gen          <- Yaml.decodeFileThrow genesis :: IO GenesisBlock
     genState     <- either (die . fromEvalError) pure $
                         evalBlock Rad.txEval Rad.pureEnv gen
-    stStore      <- StateStore.fromStateM genState
+    stStore      <- newHashStoreIO
+    storeHashContent stStore genState
```

Comment:
Minor: in the future you might think about adding a `newEmptyHashStoreIO` and a `newHashStoreIO`, where the former will be exactly what you have now, and the latter would allow you to take an initial value as input, so that you could theoretically write the above as:

```
stStore <- newHashStoreIO genState
```

Reply:
Yes, makes a lot of sense to add this in the future.